python/ray/services.py

```diff
@@ -1043,6 +1061,37 @@ def start_raylet(redis_address,
     return process_info
 
 
+def build_java_worker_command(include_java,
+                              java_classpath,
+                              redis_address,
+                              plasma_store_name,
+                              raylet_name):
+    """This method assembles java worker command.
+    """
+    if not include_java:
+        return ""
+
+    if java_classpath is None:
+        raise Exception("java_classpath must not be "
+                        "None if include_java is True.")
+
+    command = ("java -classpath %s " % java_classpath)
```

Comment:
For string formatting, let's use `"java -classpath {} ".format(java_classpath)` (same with all the other places).

This is for consistency with the rest of the codebase.